Add missing include of string.h for memset in spf_utils.c
https://github.com/shevek/libspf2/issues/41

diff --git a/src/libspf2/spf_utils.c b/src/libspf2/spf_utils.c
index 54878fa..892cea5 100644
--- a/src/libspf2/spf_utils.c
+++ b/src/libspf2/spf_utils.c
@@ -25,6 +25,9 @@
 #include <memory.h>
 #endif
 
+#ifdef HAVE_STRING_H
+# include <string.h>       /* memset */
+#endif
 
 
 #include "spf.h"
